docs/book/src/introduction.md

```diff
@@ -9,14 +9,21 @@
 
 ## What is the Cluster API Provider IBM Cloud
 
-
 The [Cluster API](https://github.com/kubernetes-sigs/cluster-api) brings declarative, Kubernetes-style APIs to cluster creation, configuration and management.
 
 The API itself is shared across multiple cloud providers allowing for true IBM Cloud
 hybrid deployments of Kubernetes.  It is built atop the lessons learned from
 previous cluster managers such as [kops](https://github.com/kubernetes/kops) and
 [kubicorn](http://kubicorn.io/).
 
+## CAPIBM Supported Infrastructure-as-a-Service (IaaS)
+
+<p align="center">
+<img src="../images/ibm-cloud-iaas.png" alt="Supported IBM Cloud IaaS">
+</p>
+
+Currently, the CAPIBM project exclusively facilitates the deployment of Kubernetes (K8s) clusters solely on two IBM infrastructure offerings, namely IBM VPC (Virtual Server Instances) and IBM PowerVS.
```

Comment:
please add the hyperlink to both vpc and powervs

Reply:
Done.